conformance/packages/dns-test/src/templates/hickory.resolver.toml.jinja

```diff
@@ -4,4 +4,4 @@ group = "nogroup"
 [[zones]]
 zone = "."
 zone_type = "Hint"
-stores = { type = "recursor" , roots = "/etc/root.hints" {% if use_dnssec %}, dnssec_policy.ValidateWithStaticKey.path = "/etc/trusted-key.key" {% else %}, dnssec_policy = "ValidationDisabled" {% endif %} , recommended_filters = false }
+stores = { type = "recursor" , roots = "/etc/root.hints" {% if use_dnssec %}, dnssec_policy.ValidateWithStaticKey.path = "/etc/trusted-key.key" {% else %}, dnssec_policy = "ValidationDisabled" {% endif %} , query_allow = ["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"] }
```

Comment:
Naming: is `query_allow` the best name? Why do we need the `query_` prefix? I seem to recall we do have other allow/deny lists...

Reply:
I think it is a clearer than just allow/deny = ..., but maybe not?  As for other prefixes, the other options I thought of seem less intuitive. (referral_allow, outbound_allow, nameserver_allow, etc.)

outbound_query_allow and outbound_query_deny is descriptive, I think understandable, but also very verbose.